.github/workflows/bench_test.yml

```diff
@@ -1,9 +1,6 @@
 name: bench_test
 
 on:
-  push:
```

Comment:
I think we should test when push to main, because the codes might change after merging to main.

Reply:
Directly pushing to the main branch is forbidden; only pull requests are allowed. Code can only be merged into the main branch once all continuous integration checks have passed.